demisto_sdk/commands/lint/linter.py

```diff
@@ -189,6 +193,7 @@ def _gather_facts(self, modules: dict) -> bool:
         # Facts for python pack
         if self._pkg_lint_status["pack_type"] == TYPE_PYTHON:
             if self._facts["docker_engine"]:
+                self._update_support_level()
```

Comment:
don't we need to update the support level even if docker_engine does not exist?

Reply:
No, as lint runs only on dockers.